specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/PrivateStores/{PrivateStoreId}/Offers": {
```

Comment:
>Offers [](start = 69, length = 6)

Use camelCase for resource type names

Reply:
What do you mean ? offers with lower case?

Reply:
done